docs/guides/transportation.mdx

```diff
@@ -49,6 +63,210 @@ are currently the most developed part of the `transportation` schema. Read more
+
+These examples use data properties to filter the data in useful ways using DuckDB.
+<Tabs>
+<TabItem value="Query by class" label="Query by class" default>
+The `class` column can be used to pull out subsets of the road data. Similarly, you could use subtype to select only water, rail, or road features. This example extracts only the **parking_aisle** features within the bounding box.
+<QueryBuilder query={ParkingAisles}></QueryBuilder>
+</TabItem>
+<TabItem value="Query by routes" label="Query by routes" default>
+You might be interested in a network of roads, such as a US Interstate. These can be extracted using the routes column and either using the network and ref properties and/or the wikidata tag to identify the route.
+
+This example extracts all the roads that are part of US I-5. To get all US Interstates simply remove `AND routes[1].ref = '5'` from the query.
+<QueryBuilder query={Routes}></QueryBuilder>
+</TabItem>
+</Tabs>
+
+### Querying by properties in Athena
+Athena can allow for faster querying of the transportation layer than DuckDB given the size of the data. These examples are designed for Athena, but could be reworked for DuckDB with some tweaking.
+
+<Tabs>
+<TabItem value="Query by speed limit" label="Query by speed limit" default>
+To properly return a linear referenced feature like a speed limit, we will need to query all the possible values of the feature as the queried value may only exist on one portion of the line. In this example, we're extracting roads with any speed limit `max_speed` value of **27** and unit of **mph** using the any_match function.
+
+This same general query would also work for querying other similar columns such as `prohibited_transitions` and `access_restrictions`.
+<QueryBuilder query={SpeedLimits}></QueryBuilder>
+</TabItem>
+<TabItem value="Select connecting segments" label="Select connecting segments" default>
+With the `connecting_segments` column it is simple to query for all features that connect with a particular segment without the need for a spatial query.
+
+This example selects all the segments that that connect to the example id.
+<QueryBuilder query={ConnectingSegments}></QueryBuilder>
+</TabItem>
+</Tabs>
+
+
+## Tools and libraries
+
+### transportation-splitter
+
+| ![Overture Splitter](/img/transportation/splitter_concept.svg) | 
+|:--:| 
+| *Conceptual diagram of the splitter tool output. The numbers follwing 1234@ represent start_lr and end_lr values.* | 
+
+The [transportation-spitter tool](https://github.com/OvertureMaps/transportation-splitter) transforms Overture road data into simpler sub segments. It will divide features at each connector point and at each change of a [scoped based property](/schema/concepts/scoping-rules/). Depending on your needs and map stack, the resulting dataset may be easier to manipulate than the original Overture data as each segment will only have connections at either end and have one set of properties for its entire length.
```

Comment:
```suggestion
The [transportation-spitter tool](https://github.com/OvertureMaps/transportation-splitter) transforms Overture road data into simpler sub segments. It will optionally divide features at each connector point and at each change of a [scoped based property](/schema/concepts/scoping-rules/), depending on configuration. Depending on your needs and map stack, the resulting dataset may be easier to manipulate than the original Overture data as each segment will only have connections at either end and have one set of properties for its entire length.
```